src/commons/BrowserUtils.ts

```diff
@@ -53,9 +60,9 @@ export namespace BrowserUtils {
   /**
    * Inject a snippet of JavaScript into the page
    * for execution in the context of the currently selected frame
-   * @param script - js script to execute
+   * @param script - js script to execute in string format
    */
-  export function executeScript(script: string): string {
+  export function execute(script: string): string {
```

Comment:
you should add optional arguments parameter

Reply:
since we are passing the full script everywhere, and for the sake of simplicity, I left it without arguments

Reply:
ok